simcon_project/templates/researcher_base.html

```diff
@@ -18,6 +18,7 @@
         </div>
         <ul class="nav navbar-nav">
           <li>{% block nav-researcherView %}<a href="{% url 'ResearcherView' %}">Home</a>{% endblock %}</li>
+          <li>{% block nav-templateView %}<a href="{% url 'management:main' %}">Templates</a>{% endblock %}</li>
```

Comment:
You should be able to just say "MainView".. but this seems to work!

Reply:
I used the alias since I thought management:main would be more clear than just MainView